sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyProperties.cs

```diff
@@ -55,70 +55,67 @@ public KeyProperties(Uri id)
         private KeyAttributes _attributes;
 
         /// <summary>
-        /// Name of the key.
+        /// Gets the name of the key.
         /// </summary>
         public string Name { get; internal set; }
 
         /// <summary>
-        /// Key identifier.
+        /// Gets the key identifier.
         /// </summary>
         public Uri Id { get; internal set; }
 
         /// <summary>
-        /// Vault base URL.
+        /// Gets the Key Vault base <see cref="Uri"/>.
         /// </summary>
         public Uri VaultEndpoint { get; internal set; }
 
         /// <summary>
-        /// Version of the key.
+        /// Gets the version of the key.
         /// </summary>
         public string Version { get; internal set; }
 
         /// <summary>
-        /// Set to true if the key's lifetime is managed by key vault. If this
-        /// is a key backing a KV certificate, then managed will be true.
+        /// Gets a value indicating whether the key's lifetime is managed by Key Vault.
+        /// If this key is backing a Key Vault certificate, the value will be true.
         /// </summary>
         public bool Managed { get; internal set; }
 
         /// <summary>
-        /// A dictionary of tags with specific metadata about the key.
+        /// Gets a dictionary of tags with specific metadata about the key.
         /// </summary>
         public IDictionary<string, string> Tags => LazyInitializer.EnsureInitialized(ref _tags);
 
         /// <summary>
-        /// Specifies whether the key is enabled and useable for cryptographic operations.
+        /// Gets or sets a value indicating whether the key is enabled and useable for cryptographic operations.
         /// </summary>
         public bool? Enabled { get => _attributes.Enabled; set => _attributes.Enabled = value; }
 
         /// <summary>
-        /// Identifies the time (in UTC) before which the key must not be used for cryptographic operations.
+        /// Gets or sets a <see cref="DateTimeOffset"/> of when the key will be valid and cannot be used.
```

Comment:
~~~
Gets or sets the <see cref="DateTimeOffset"/> when the key will be valid for use and can be used for cryptographic operations.